CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+- Add ability to extract trace headers (traceparent and tracestate) from Xmidt headers, JSON, and msgpack and propagates the trace context accordingly [#115](https://github.com/xmidt-org/candlelight/pull/115)
```

Comment:
@schmidtw I know we don't need to update the Changelogs anymore, but is it still good practice?

Reply:
It's probably better to update when the file is there.